src/index.js

```diff
@@ -71,7 +71,7 @@ export default function replace(options = {}) {
 			if (!hasReplacements) return null;
 
 			let result = { code: magicString.toString() };
-			if (options.sourceMap !== false && options.sourcemap !== false)
```

Comment:
I believe the original code is actually correct here as source maps are *always* generated *unless* one of the two variants of this option set them to `false`. Nevertheless I am currently struggling to write a test where using this option makes any difference.

Reply:
Sorry for getting back to you only now. I now realise that I was missing the fact that the default behaviour is to generate the sourceMap and that the default value for both options is `undefined`.

If anyone stumble upon this issue, @lukastaegert is correct and this is the table of truth for extra clarity:

| generate? | sourceMap | undefined | true | false |
|:---------:|:---------:|:---------:|:----:|:-----:|
| sourcemap |  |  |  |  |
| undefined |  | 1 | 1 | 0 |
| true |  | 1 | 1 | 0 |
| false |  | 0 | 0 | 0 |